torch_harmonics/convolution.py

```diff
@@ -275,7 +190,7 @@ def _precompute_convolution_tensor_s2(
     else:
         quad_weights = 2.0 * torch.pi * torch.from_numpy(win).float().reshape(-1, 1) / nlon_in
     out_vals = _normalize_convolution_tensor_s2(
-        out_idx, out_vals, in_shape, out_shape, kernel_shape, quad_weights, transpose_normalization=transpose_normalization, merge_quadrature=merge_quadrature
+        out_idx, out_vals, in_shape, out_shape, kernel_size, quad_weights, transpose_normalization=transpose_normalization, merge_quadrature=merge_quadrature
```

Comment:
should we name it kernel_sizes, to make it more clear that you can also do anisotropic kernels?